LibVLCSharp.Uno/LibVLCSharp.Uno.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/2.0.54">
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <Nullable>enable</Nullable>
+    <TargetFrameworks>xamarinios10;monoandroid81;monoandroid90</TargetFrameworks>
+    <!-- Ensures the .xr.xml files are generated in a proper layout folder -->
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
+    <Authors>VideoLAN</Authors>
+    <Owners>VideoLAN</Owners>
+    <Version>0.1.1-alpha</Version>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <Description>VLC MediaPlayerElement for Uno Platform (UWP, Android, iOS)</Description>
+    <PackageTags>libvlc;vlc;videolan;video;audio;player;media;mediaplayer;mediaelement;mediaplayerelement;codec;ffmpeg;graphics;ios;android;windows;cross-platform;uno</PackageTags>
+    <PackageProjectUrl>https://github.com/libvlcsharp/libvlcsharp.uno</PackageProjectUrl>
+    <PackageId>LibVLCSharp.Uno</PackageId>
+    <PackageLicenseExpression>LGPL-2.1-or-later</PackageLicenseExpression>
+  </PropertyGroup> 
+  <ItemGroup Condition="'$(TargetFramework)'=='xamarinios10' or '$(TargetFramework)'=='monoandroid81' or '$(TargetFramework)'=='monoandroid90'">
+    <PackageReference Include="Uno.UI" Version="2.0.512-dev.3565" />
```

Comment:
Do you need that pre-release version specifically?

Reply:
No, I'm taking the last pre-release of version 2, waiting a stable version.